노현호/정렬/K번째수.js

```diff
@@ -0,0 +1,16 @@
+function solution(array, commands) {
+  let a, b, c;
+  var answer = [];
+
+  commands.forEach((command) => {
+    a = command[0];
+    b = command[1];
+    c = command[2];
```

Comment:
이렇게 구조분해할당 할 수 있을 것 같아요!
```suggestion
  commands.forEach(([a, b, c]) => {
```